walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,110 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.lang.Math;
 
 public final class FindMeetingQuery {
+    private static final TimeRange START_OF_DAY_TIMERANGE = TimeRange.fromStartDuration(
+        TimeRange.START_OF_DAY, 0);
+    private static final TimeRange END_OF_DAY_TIMERANGE = TimeRange.fromStartDuration(
+        TimeRange.END_OF_DAY + 1, 0);
+
+  // returns time ranges that will accomadate meeting request
```

Comment:
Funky indents

Reply:
I did +4 spaces for line continuation but maybe I am incorrect for methods.